examples/vnet/kubernetesvnet-vmss.json

```diff
@@ -0,0 +1,47 @@
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes",
+      "orchestratorRelease": "1.10",
+      "kubernetesConfig": {
+          "clusterSubnet": "172.16.3.0/24",
+          "privateCluster": {
+              "enabled": true
+          }
+      }
+    },
+    "masterProfile": {
+      "count": 1,
+      "dnsPrefix": "newprivatecluster",
+      "vmSize": "Standard_D2_v2",
+      "vnetSubnetId": "/subscriptions/931980ec-6458-48c2-9668-354453011dac/resourceGroups/RG-NAME/providers/Microsoft.Network/virtualNetworks/VNET-NAME/subnets/SUBNET-NAME",

```

Comment:
Let's make the vnetSubnetId `"/subscriptions/SUB_ID/resourceGroups/RG_NAME/providers/Microsoft.Network/virtualNetworks/VNET_NAME/subnets/SUBNET_NAME"` to be consistent with other example files (and remove the hardcoded sub ID)

Reply:
done.